### ISSUE TITLE
Installation writes to standard error when no error occurs

### ISSUE THREAD

**Issue by mark-raymond** [sass/node-sass#1801]
Since node-sass 3.11.3, installation writes to standard error, even when no error occurs. This is a problem on some build servers (at least including TeamCity), as output to standard error fails the build.

Reproduction steps:

- Copy packages.json from [attached zip](https://github.com/sass/node-sass/files/589651/node-sass-bug.zip) to an empty folder
- Run `npm install --loglevel=error >npm.log 2>npm.err`

Expected result:

- Nothing written to npm.err

Actual result:

- See npm.err in [attached zip](https://github.com/sass/node-sass/files/589651/node-sass-bug.zip)

Version information:

- NPM version (`npm -v`): 3.10.9
- Node version (`node -v`): 4.4.7
- Node Process (`node -p process.versions`): 
```
{ http_parser: '2.5.2',
  node: '4.4.7',
  v8: '4.5.103.36',
  uv: '1.8.0',
  zlib: '1.2.8',
  ares: '1.10.1-DEV',
  icu: '56.1',
  modules: '46',
  openssl: '1.0.2h' }
```
- Node Platform (`node -p process.platform`): win32
- Node architecture (`node -p process.arch`): x64
- node-sass version (`node -p "require('node-sass').info"`):
```
node-sass       3.12.2  (Wrapper)       [JavaScript]
libsass         3.3.6   (Sass Compiler) [C/C++]
```
- npm node-sass versions (`npm ls node-sass`):
```
C:\Work\node-sass-bug
`-- node-sass@3.12.2
```

**Comment by martincostello** [comment]
I'm still having this issue in AppVeyor through PowerShell with 3.12.5 ([build](https://ci.appveyor.com/project/martincostello/website/build/1.0.173#L362)).

I've worked around it by fixing ```node-sass``` going back to [3.10.1](https://github.com/martincostello/website/pull/99/commits/311aff26fddcd221157a23d77d7f37e7bf682174).

**Comment by saper** [comment]
This an issue with PowerShell, it assumes that if anything got posted on standard error the command failed. UNIX-like systems need an error code for that.

**Comment by martincostello** [comment]
But why is node-sass now outputting to standard error in newer versions when there is nothing apparently wrong? It only seems to be downloading a file.

**Comment by xzyfer** [comment]
We moved our logging to the same logging library as npm, which by default logs to stderr. It's caused some issue so we're reverting it shortly.

**Comment by xzyfer** [comment]
@martincostello released v3.13.0 which should now only log to stderr if there is an error

**Comment by martincostello** [comment]
@xzyfer Fixed now - thank you :)
